forms/models/element.js

```diff
@@ -247,10 +249,9 @@ define(function (require) {
       }
 
       result = Backbone.Model.prototype.set.call(this, attrs, options);
-
-      if ('value' in attrs) {
-        if (!options || !options.hasOwnProperty('value') || options.validate) {
-          this.isValid();
+      if ('blob' in attrs || 'value' in attrs) {
+        if (!options || (!options.hasOwnProperty('value') || !options.hasOwnProperty('blob')) || options.validate) {
```

Comment:
I've reviewed the source code for Backbone and our reasons for overriding `#set()`, and I believe having `!options.hasOwnProperty('value')` in the original condition was unnecessary, so I believe having `(!options.hasOwnProperty('value') || !options.hasOwnProperty('blob'))` in this PR is also unnecessary.

I'm sorry to be such a pain, but could please re-test without this condition and see if we can remove it before proceeding further?